forem/forem
# Point changes to inbox_type and inbox_guidelines attributes to users_settings table

## Description
This PR point changes to `inbox_type` and `inbox_guidelines` within the `Extensions` tab to the `users_settings` table. This PR shall be merged into [PR 13904](https://github.com/forem/forem/pull/13904).

## TO-DO list for this PR
- [x] Update relevant attributes in the `customization` and `profile` tabs to use the new `users_settings` table 
- [x] Go through the TO-DO's on the application 
- [x] Check for other instances where these fields are being used


## What type of PR is this? (check all applicable)



## QA Instructions, Screenshots, Recordings
1. Sign into `localhost`; in rails console, pull up the same user that you are signed in as.
2. In rails console, run `user.setting` and take note the values for `inbox_type` and `inbox_guidelines`. Also run `user.inbox-type` and `user.inbox_guidelines` and note that the values match those in the `notification_setting` record. 
3. In the app, navigate to `localhost:xxxx/settings/extensions`; toggle the Inbox Privacy setting, and enter some content in the Open inbox guidelines/instructions field. Click Save Connect Settings. You should be redirected to the Profile tab.
4.  In the console, reload your user (`user.reload`) and re-run `user.setting`, `user.inbox_type` and `user.inbox-guidelines`. Now, the `setting` values should differ from the `user.` values, as the `users` record was left unchanged, but the `settings` record was changed accordingly. 


### UI accessibility concerns? NONE (backend changes)

## Added tests?

      have not been included_
- [X] Not yet; will write after merging into [PR 13904](https://github.com/forem/forem/pull/13904)

## [Forem core team only] How will this change be communicated?
